src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -278,8 +281,7 @@ private void DefineCategoriesFullCategoryName(ObservableCollection<NodeCategoryV
         {
```

Comment:
I think a better name would be `DefineFullCategoryNames`, and then make it `static`.